evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -390,3 +390,23 @@ def test_graph_partial_dependence_multiclass(logistic_regression_multiclass_pipe
     msg = "Class wine is not one of the classes the pipeline was fit on: class_0, class_1, class_2"
     with pytest.raises(ValueError, match=msg):
         graph_partial_dependence(pipeline, X, features='alcohol', class_label='wine')
+
+
+def test_partial_dependence_percentile_errors(logistic_regression_binary_pipeline_class):
+    # random_col will be 5% 0, 95% 1
+    X = pd.DataFrame({"A": [i % 3 for i in range(1000)], "B": [(j + 3) % 5 for j in range(1000)], "random_col": [0 if i < 50 else 1 for i in range(1000)]})
+    y = pd.Series([i % 2 for i in range(1000)])
+    pipeline = logistic_regression_binary_pipeline_class(parameters={"Logistic Regression Classifier": {"n_jobs": 1}})
+    pipeline.fit(X, y)
+    with pytest.raises(ValueError, match="Feature 'random_col' is mostly one value, 1, and cannot be"):
+        partial_dependence(pipeline, X, features="random_col", grid_resolution=20)
+    with pytest.raises(ValueError, match="Feature 'random_col' is mostly one value, 1, and cannot be"):
+        partial_dependence(pipeline, X, features="random_col", percentiles=(0.01, 0.955), grid_resolution=20)
+    with pytest.raises(ValueError, match="Feature '2' is mostly one value, 1, and cannot be"):
+        partial_dependence(pipeline, X, features=2, percentiles=(0.01, 0.955), grid_resolution=20)
```

Comment:
Hmmm, that'll be a yike from me.  I guess it's possible for X to not have a good feature name in that column, eh?  Is it worth it trying to return the string feature name?  If not, no biggie.  We're all responsible adults I guess, lol

Reply:
Yep, I'll return the name then!